<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPackageBuild</class>
 <widget class="QDialog" name="DlgPackageBuild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Package</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Platform</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radDesktop">
        <property name="text">
         <string>Desktop</string>
        </property>
        <property name="icon">
         <iconset resource="../../../data/HyEditor.qrc">
          <normaloff>:/icons16x16/Build-Desktop.png</normaloff>:/icons16x16/Build-Desktop.png</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radBrowser">
        <property name="text">
         <string>Browser</string>
        </property>
        <property name="icon">
         <iconset resource="../../../data/HyEditor.qrc">
          <normaloff>:/icons16x16/Build-Browser.png</normaloff>:/icons16x16/Build-Browser.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output Location:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>./bin/package</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDirName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Assets</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>Loose Files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Packed Assets</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Emscripten</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="2">
           <widget class="QCheckBox" name="chkBrowserDebug">
            <property name="text">
             <string>Debug Build</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>SDK Location:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="txtEmscriptenSdk"/>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="btnEmscriptenSdkBrowse">
            <property name="text">
             <string>Browse..</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblError">
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../data/HyEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgPackageBuild</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgPackageBuild</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
